vowpalwabbit/parse_example.cc

```diff
@@ -292,7 +292,7 @@ class TC_parser
             spelling_v.push_back(_index);
             spelling_v.push_back('_');
           }
-          push_many(spelling_v, spelling_strview.begin(), spelling_strview.size());
+          spelling_v.insert(spelling_v.begin(), spelling_strview.begin(), spelling_strview.end());
```

Comment:
inserting to the beginning of spelling_v?

Reply:
Ah it should be end thanks!